examples/browser/index.html

```diff
@@ -0,0 +1,15 @@
+<html><head>
```

Comment:
The following makes it a bit more standards compliant and gets rid of a warning in Firefox (although Firefox doesn't support WebMIDI yet):

```suggestion
<!DOCTYPE html>
<html><head>
    <meta charset="utf-8"/>
```

Reply:
Also adding:
```html
<meta http-equiv="X-UA-Compatible" content="IE=edge">
```
So IE/Edge don't throw less of a fit, not that they support WebMIDI at the moment either...